_shared/project/Makefile

```diff
@@ -96,9 +96,9 @@ $(call help,make {{ target }},"run the unit tests in Python {{ python_version|py
 $(call help,make coverage,"run the tests and print the coverage report")
 coverage: python
 {% if cookiecutter._directory in ['pyapp', 'pyramid-app'] %}
-	@pyenv exec tox -qe 'tests,coverage'
+	@pyenv exec tox -qe 'clean,tests,coverage'
 {% else %}
-	@pyenv exec tox --parallel -qe 'tests,py{{ "{" }}{{ python_versions()|rest|pyformat(PyFormats.MAJOR_MINOR_FMT)|separator(",") }}{{ "}" }}-tests,coverage'
+	@pyenv exec tox --parallel -qe 'clean,tests,py{{ "{" }}{{ python_versions()|rest|pyformat(PyFormats.MAJOR_MINOR_FMT)|separator(",") }}{{ "}" }}-tests,coverage'
```

Comment:
Add the `clean` environment to `make coverage` so that it deletes the coverage files before re-running the tests